runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingModeExecutionContext.java

```diff
@@ -528,7 +535,8 @@ public void start(
               inputDataWatermark,
               processingTime,
               outputDataWatermark,
-              synchronizedProcessingTime);
+              synchronizedProcessingTime,
+              null);
```

Comment:
Prefer a noop lambda to making this `@Nullable`

Reply:
Good suggestion. done